backend/users/urls.py

```diff
@@ -1,7 +1,9 @@
 from django.urls import path
-
 from users.views import UserRegisterView
 
+
+app_name = "users"
```

Comment:
Convention is to use singular form
```suggestion
app_name = "user"
```